sycl/test/check_device_code/const_ref_image_accessor.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: %clangxx -S -fsycl -fsycl-device-only %s
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsycl-device-only %s -o /dev/null
```
If we don't check the emitted code, `-S` is not needed as well.